packages/core/src/command/patient-import/patient-import-shared.ts

```diff
@@ -56,12 +56,13 @@ export const PatientImportCsvHeaders = [
   "addressline2",
   "phone1",
   "email1",
-  "phone2",
-  "email2",
+  // "phone2",
+  // "email2",
```

Comment:
We need a better way to process those so we accept small variations on column and names, unless we really want to be strict about it - currently failing for the current batch of imports.

Regardless, it would be good to have a reference CSV on the repository, that we could point customers to, so they can download it and populate off of that reference file.

Reply:
Will these be included for our final release or no?

Reply:
We either adjust these to the upcoming two bulk imports or we update each file to match what we have here.

Right now, if the cx provides a CSV w/o phone2 column, it breaks.